presto-hive/src/main/java/io/prestosql/plugin/hive/security/SqlStandardAccessControlMetadata.java

```diff
@@ -55,20 +56,25 @@ public SqlStandardAccessControlMetadata(SemiTransactionalHiveMetastore metastore
     @Override
     public void createRole(ConnectorSession session, String role, Optional<HivePrincipal> grantor)
     {
-        // roles are case insensitive in Hive
-        if (RESERVED_ROLES.contains(role)) {
-            throw new PrestoException(ALREADY_EXISTS, "Role name cannot be one of the reserved roles: " + RESERVED_ROLES);
-        }
+        checkRoleIsNotReserved(role);
         metastore.createRole(role, null);
     }
 
     @Override
     public void dropRole(ConnectorSession session, String role)
     {
-        // roles are case insensitive in Hive
+        checkRoleIsNotReserved(role);
```

Comment:
This change doesn't match the commit message.

Maybe split into two commits?

Reply:
Will update commit message